README.md

```diff
@@ -300,7 +300,7 @@ Additionally, an [example AML application](/examples/aml/main.go) is provided in
 The profile retrieval example can be found in the [examples folder](examples).
 
 * Change directory to the profile example folder: `cd examples/profile`
-* On the [Yoti Dashboard](https://www.yoti.com/dashboard/applications):
+* On the [Yoti Hub](https://hub.yoti.com/applications):
```

Comment:
> Same as comment on https://github.com/getyoti/yoti-python-sdk/pull/73

Should we drop `/applications` @echarrod ?

Reply:
Yes, the default is `/organisations/<theorgid>/applications`, so since org id is dynamic, we should drop /applications 👍 . `/applications` will take you to your personal applications, which might be confusing.